src/MemoryTracing/MemoryTracingUtils.cpp

```diff
@@ -197,24 +190,62 @@ ErrorMessageOr<void> UpdateProcessMemoryUsageFromProcessStat(
   return outcome::success();
 }
 
+ErrorMessageOr<void> UpdateProcessMemoryUsageFromProcessStatus(
+    std::string_view status_content, ProcessMemoryUsage* process_memory_usage) {
```

Comment:
As far as I can see this functions only calls `set_rss_anon_kb` from `ProcessMemoryUsage`. Have you considered returning thos one value instead of using an out-parameter? I mean like that: `ErrorMessageOr<int64_t> ExtractRssAnonFromProcessStatus(std::string_view status_content);`?

Reply:
Changed accordingly. Thanks for the suggestion!